fastparquet/test/test_json.py

```diff
@@ -25,15 +26,15 @@ def _clear_cache():
 
 
 @pytest.mark.parametrize(
-    "data",
+    "data,has_float64",
     [
-        None,
-        [1, 1, 2, 3, 5],
-        [1.23, -3.45],
-        [np.float64(0.12), np.float64(4.56)],
-        [[1, 2, 4], ["x", "y", "z"]],
-        {"k1": "value", "k2": "à/è", "k3": 3},
-        {"k1": [1, 2, 3], "k2": [4.1, 5.2, 6.3]},
+        (None, False),
+        ([1, 1, 2, 3, 5], False),
+        ([1.23, -3.45], False),
+        ([np.float64(0.12), np.float64(4.56)], True),
```

Comment:
not sure what to do about this but rapidjson.dumps produces invalid json for:

```python
>>> rapidjson.dumps(np.float64(0.1))
'np.float64(0.1)'
```

Reply:
I don't suppose there's a good reason rapidjson should know what to do with that